README.md

```diff
@@ -36,14 +36,39 @@ mockServer.AddVerification(
     ))
 ```
 
-Expectation Defaults:
+Create a verification sequence example:
+```
+mockServer := mockclient.Client{
+    T: t,
+    BaseURL: os.Getenv("MOCKSERVER_HOST"),
+}
+
+mockServer.AddVerificationSequence(
+    mockclient.CreateVerification(
```

Comment:
Not having followed the evolution of the mockServer, it is unclear to me what is being verified here?
There is a temporal condition `WhenRequestPath("/a")` but nothing expressing what is to be the expectation for the verification.

Reply:
nvm I got it from the README text later.